packit_service_fedmsg/consumer.py

```diff
@@ -36,6 +37,23 @@
 }
 KOJI_TOPICS = {"org.fedoraproject.prod.buildsys.task.state.change"}
 
+PUSH_TOPIC = "org.fedoraproject.prod.git.receive"
+
+
+def specfile_changed(body) -> bool:
+    """
+    Does the commit contain specfile change?
+    :param body: message body
+    :return: bool
+    """
+    files = reduce(
+        lambda val, key: val.get(key) if val else None,
+        ["commit", "stats", "files"],
+        body,
+    )
+    file_names = list(files.keys()) if files else []
```

Comment:
tbh I don't think you need to create list, since `.keys()` returns iterable and you just use it in generator expression below